src/components/Carousel/CarouselPanel.js

```diff
@@ -73,6 +102,11 @@ CarouselPanel.propTypes = {
    */
   imageUrl: PropTypes.string.isRequired,
 
+  /**
+   * Callback for keyboard interaction on an active panel.
+   */
+  onKeyboardInteraction: PropTypes.func,
```

Comment:
You could require this here and mark it as private as it should not be messed with by end users